strawberry/types/info.py

```diff
@@ -36,15 +45,15 @@ def schema(self) -> "Schema":
         return self._raw_info.schema._strawberry_schema  # type: ignore
 
     @property
-    def field_nodes(self) -> List[FieldNode]:  # deprecated
+    def field_nodes(self) -> Collection[FieldNode]:  # deprecated
```

Comment:
Not sure why `GraphQLResolveInfo.field_nodes` is typed as `Collection[FieldNode]` in graphql-core 🤔 It appears whenever they construct a `GraphQLResolveInfo` instance their field nodes are lists.